alerta-covid/app/models/place.rb

```diff
@@ -1,4 +1,21 @@
 class Place < ApplicationRecord
   belongs_to :area
   validates :name, :area, presence: true
+
```

Comment:
you can add after_save: generate_qr_code, so for each places created, you generate its qr code

Reply:
In the controller I call this method each time show the place